src/test/ui/issue-22644.stderr

```diff
@@ -1,24 +1,66 @@
 error: `<` is interpreted as a start of generic arguments for `usize`, not a comparison
-  --> $DIR/issue-22644.rs:16:33
+  --> $DIR/issue-22644.rs:16:31
    |
-16 |     println!("{}", a as usize < b);
-   |                               - ^ interpreted as generic argument
+16 |     println!("{}", a as usize < long_name);
+   |                               ^----------
    |                               |
+   |                               interpreted as generic arguments
    |                               not interpreted as comparison
    |
 help: if you want to compare the casted value then write:
-   |     println!("{}", (a as usize) < b);
+   |     println!("{}", (a as usize) < long_name);
 
 error: `<` is interpreted as a start of generic arguments for `usize`, not a comparison
   --> $DIR/issue-22644.rs:17:33
    |
-17 |     println!("{}", a as usize < 4);
-   |                               - ^ interpreted as generic argument
+17 |     println!("{}{}", a as usize < long_name, long_name);
+   |                                 ^---------------------
+   |                                 |
+   |                                 interpreted as generic arguments
+   |                                 not interpreted as comparison
+   |
+help: if you want to compare the casted value then write:
+   |     println!("{}{}", (a as usize) < long_name, long_name);
+
+error: `<` is interpreted as a start of generic arguments for `usize`, not a comparison
+  --> $DIR/issue-22644.rs:18:31
+   |
+18 |     println!("{}", a as usize < 4);
+   |                               ^--
    |                               |
+   |                               interpreted as generic arguments
    |                               not interpreted as comparison
    |
 help: if you want to compare the casted value then write:
    |     println!("{}", (a as usize) < 4);
 
-error: aborting due to 2 previous errors
+error: `<` is interpreted as a start of generic arguments for `usize`, not a comparison
+  --> $DIR/issue-22644.rs:20:31
+   |
+20 |     println!("{}{}", a: usize < long_name, long_name);
+   |                               ^---------------------
+   |                               |
+   |                               interpreted as generic arguments
+   |                               not interpreted as comparison
```

Comment:
Will look at the code this afternoon to see what change would be needed for this suggestion to be implemented, but I feel the output should be:

```
   |
20 |     println!("{}{}", a: usize < long_name, long_name);
   |                               ^ -------------------- interpreted as generic arguments
   |                               |
   |                               not interpreted as comparison
```